src/rust/lib/graph-editor/src/component/visualization.rs

```diff
@@ -0,0 +1,332 @@
+//! This module defines the visualization widgets and related functionality.
+//!
+//! At the core of this functionality is the `Visualisation` that takes in data and renders an
+//! output visualisation which is displayed in a `Container`. The `Container` provides generic UI
+//! elements that facilitate generic interactions, for example, visualisation selection. The
+//! `Container` also provides the FRP API that allows internal interaction with the
+//! `Visualisation`. Data for a visualisation has to be provided wrapped in the `Data` struct.
+//!
+use crate::prelude::*;
+
+use crate::frp;
+
+use ensogl::display::DomSymbol;
+use ensogl::display::object::class::Object;
+use ensogl::display::object::class::ObjectOps;
```

Comment:
You should not need this. It is imported by prelude.

Reply:
The prelude doesn't seem to be enough. I seem to need `use ensogl::display::traits::*;`, but the takes care of `use ensogl::display::object::class::Object;` and `use ensogl::display::object::class::ObjectOps;` .

Reply:
Please add then `pub use ensogl::display::traits::*;` to prelude IN THIS CRATE only.

Reply:
Done.